cadctl/cmd/cluster-missing/cluster-missing.go

```diff
@@ -85,6 +85,13 @@ func run(cmd *cobra.Command, args []string) error {
 
 	fmt.Printf("ClusterExternalID is: %s\n", externalClusterID)
 
+	cloudProvider, err := ocmClient.GetCloudProvider(externalClusterID)
```

Comment:
In case you go with the below suggestion

```suggestion
	cloudProvider, err := ocmClient.GetCloudProviderID(externalClusterID)
```